src/alfasim_score/converter/alfacase/base_operation.py

```diff
@@ -97,6 +172,16 @@ def build_nodes(self) -> List[NodeDescription]:
                     temperature=operation_data["flow_initial_temperature"],
                     pressure=operation_data["flow_initial_pressure"],
                     split_type=MassInflowSplitType.Pvt,
```

Comment:
If you are using `split_type=MassInflowSplitType.Pvt,`, you don't need to define `volume_fraction`and `mass_fraction`.
`mass_fraction` is required when using: `MassInflowSplitType.ConstantMassFraction` and `volume_fraction`, when using: `MassInflowSplitType.ConstantVolumeFraction`

Reply:
Alright. Removed.